src/celeritas/em/data/GammaNuclearData.hh

```diff
@@ -0,0 +1,90 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/em/data/GammaNuclearData.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "corecel/data/Collection.hh"
+#include "corecel/grid/NonuniformGridData.hh"
+#include "corecel/math/Quantity.hh"
+#include "celeritas/Quantities.hh"
+#include "celeritas/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Scalar data for the gamma-nuclear model.
+ */
+struct GammaNuclearScalars
+{
+    // Particle IDs
+    ParticleId gamma_id;
+
+    //! Model's maximum energy limit [MeV]
+    static CELER_CONSTEXPR_FUNCTION units::MevEnergy max_valid_energy()
+    {
+        return units::MevEnergy{5e+4};
+    }
+
+    //! Maximum energy limit for PARTICLEXS/gamma cross sections [MeV]
+    static CELER_CONSTEXPR_FUNCTION units::MevEnergy max_low_energy()
```

Comment:
This name is a little vague... maybe something like `tabulated_max_energy` or `otf_xs_threshold`?

Reply:
More parameters will be added when the high-energy cross sections are implemented. I’ll rename this accordingly once the final structure is defined.